index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>ProgramBoys</title>
```

Comment:
Nã0 gostei deste titulo
```suggestion
    <title>PedrinhoReiDelas</title>
```